File: training-2022/3.c
#include <stdio.h>
#include <stdlib.h>

int main() {
int sum = 0;

  for (int i = 1; i <= 100; i++) {
    int random_number = rand() % 101;
    printf("%d: %d\n", i, random_number);
    sum += random_number;
  }

  printf("average: %.3f\n", sum / 100.000);

  return 0;
}

File: training-2022/3.py
import random

sum = 0

for i in range(1, 101):
  random_number = random.randint(1, 100)
  print(f"{i}: {random_number}")
  sum += random_number

average = sum / 100
rounded_average = format(average, ".3f")
print(f"Average: {rounded_average}")
